### PR TITLE
Feature/update net7 tkx 14

### PR DESCRIPTION
**Pull request by TiagoFroisPereira**

### CODE REVIEW

File: src/Trakx.CryptoCompare.ApiClient.Tests/Integration/Rest/Core/ThrottledHttpClientHandlerTests.cs [hunk 1]
```diff
@@ -16,7 +15,7 @@ public async Task WaitsBetweenQueries()
             var throttleDelayMs = 200;
             var queriesCount = 5;
 
-            var configuration = ConfigurationHelper.GetConfigurationFromEnv<CryptoCompareApiConfiguration>()
+            var configuration = EnvConfigurationHelper.GetConfigurationFromEnv<CryptoCompareApiConfiguration>()
```
Comment (emanuelgaspar) [inline src/Trakx.CryptoCompare.ApiClient.Tests/Integration/Rest/Core/ThrottledHttpClientHandlerTests.cs:18]: Could we replace with AWS here? Or do you want to do it in another PR?

Comment (TiagoFroisPereira) [reply src/Trakx.CryptoCompare.ApiClient.Tests/Integration/Rest/Core/ThrottledHttpClientHandlerTests.cs:18]: To replace by Aws we need to create the secrets in AWS for CiCd, not sure they are there

Comment (TiagoFroisPereira) [reply src/Trakx.CryptoCompare.ApiClient.Tests/Integration/Rest/Core/ThrottledHttpClientHandlerTests.cs:18]: but yeah lets make it in a different PR to unlock Market-data

**Review by emanuelgaspar — APPROVED** [review]
